std/array.d

```diff
@@ -212,6 +212,13 @@ if (isNarrowString!String)
     return cast(typeof(return)) str.toUTF32;
 }
 
+///
+@safe unittest
+{
+    assert("Hello D".array == "Hello D"d);
```

Comment:
Just in case people don't know who are reading this, I would add

```d
static assert(isRandomAccessRange!string == false);
```

here, and then

```d
static assert(isRandomAccessRange!dstring == true);
```

at the end